charts/karpenter/README.md

```diff
@@ -15,7 +15,7 @@ You can follow the detailed installation instruction in the [documentation](http
 ```bash
 helm upgrade --install --namespace karpenter --create-namespace \
   karpenter oci://public.ecr.aws/karpenter/karpenter \
-  --version v0.34.0 \
+  --version 0.34.0 \
```

Comment:
Did you mean to make this change?

Reply:
Not really. I ran `make presubmit` and `make docgen`. I can revert this!

Reply:
`make docgen` would definitely produce this. I think we can drop this and take this in separately. We should also generate this when we generate a new minor version release automatically.

Reply:
I believe this is done, please let me know in case there is still something missing here